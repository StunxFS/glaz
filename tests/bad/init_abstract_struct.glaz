@[abstract]
struct Abs { }

fn main() {
	_ = Abs{};
}
