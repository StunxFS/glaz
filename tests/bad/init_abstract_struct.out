tests/bad/init_abstract_struct.glaz:5:7: error: cannot create instance of abstract struct `Abs`
     | 
   5 | 	_ = Abs{};
     | 	    ^^^
glaz error: could not compile `init_abstract_struct`, aborting due to previous error.
